### ISSUE TITLE
Solr configuration audit

### ISSUE THREAD

**Issue by hackmastera** [chemheritage/chf-sufia#352]

**Comment by hackmastera** [comment]
quoth @jrochkind "although the auto-commit is only every 4 hours!  I think our current setup may be losing data if you restart solr less than 4 hours after a change, since I think all our changes are softCommits."

**Comment by jrochkind** [comment]
maxTime is ms not seconds. So 15000 is actually 15 seconds, whole different issue, we're not losing commits. 

Prob still worth a look at solrconfig at some point, not sure if we should close this?

**Comment by hackmastera** [comment]
Great!! Yes I still think we should look at the config.

**Comment by hackmastera** [comment]
We looked at this and will defer further action for now.

However we identified some things we might want to keep in mind that could be changed or added in future, if we're either having performance issues or want to tweak for best possible performance:
- autocommit time (currently 15 sec)
- autowarming configuration (happens when a searcher is opened, i.e. on commit)
- useColdSearcher - Setting this to false blocks queries until warming is complete. what is the default value? What are the consequences of using an un-warmed searcher?
- Create a newSearcher listener event to warm facet data
